### PR TITLE
Add Flutter help menu.

### PR DESCRIPTION
**Pull request by pq**
Adds **Help > Flutter Help** menu a la Android Studio.

screen shot 2016-12-26 at 11 37 19 am

### CODE REVIEW

**Comment by pq** [comment]
@devoncarew

**Comment by devoncarew** [comment]
lgtm

File: resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -158,7 +158,12 @@
       <add-to-group group-id="ProjectViewPopupMenu" relative-to-action="CutCopyPasteGroup" anchor="before"/>
      </group>
 
-  </actions>
+    <!-- help menu -->
+    <action class="io.flutter.actions.FlutterGettingStarted" id="Flutter.FlutterHelp" text="Flutter Help">
```
Comment (devoncarew) [inline resources/META-INF/plugin.xml]: `Flutter Help` or `Flutter Plugin Help`? lgtm either way

Comment (pq) [inline resources/META-INF/plugin.xml]: 👍 

screen shot 2016-12-27 at 9 32 19 am

**Pull request merged**